products/sle-micro/main.pm

```diff
@@ -24,6 +27,20 @@ unregister_needle_tags('ENV-OFW-1');
 unregister_needle_tags('ENV-PXEBOOT-0');
 unregister_needle_tags('ENV-PXEBOOT-1');
 
+sub load_boot_from_disk_tests {
+    if (is_s390x()) {
+        loadtest 'installation/bootloader_start';
+        loadtest 'boot/boot_to_desktop';
+    } else {
+        loadtest 'microos/disk_boot';
+    }
+    loadtest 'transactional/host_config';
+    loadtest 'console/suseconnect_scc' if get_var('SCC_REGISTER');
```

Comment:
```suggestion
    loadtest 'transactional/enable_selinux' if check_var('SCC_REGISTER', 'installation');
```

don't ask me why `installation`, but it was done like this for a lot of time, but let's keep this condition and think about a more logical way to trigger this module later on.